README.md

```diff
@@ -58,24 +57,19 @@ curl -X 'POST' \
   -H 'accept: application/json' \
   -H 'Content-Type: application/json' \
   -d '{
-  "ver_config_id": "showcase-person",
+  "ver_config_id": "test-proof",
   "subject_identifier": "",
   "generate_consistent_identifier": true,
   "proof_request": {
-    "name": "BC Wallet Showcase Person",
+    "name": "Test Proof-Request",
     "version": "1.0",
     "requested_attributes": [
 
       {
-        "names": ["given_names", "family_name", "country"],
+        "names": ["attr1", "attr2", "attr3"],
         "restrictions": [
           {
-              "schema_name": "Person",
-              "issuer_did": "QEquAHkM35w4XVT3Ku5yat"
-          },
-          {
-              "schema_name": "Person",
-              "issuer_did": "RGjWbW1eycP7FrMf4QJvX8"
+              "schema_name": "test-schema",
```

Comment:
Get an error running with this trailing comma

`{"detail":[{"type":"json_invalid","loc":["body",359],"msg":"JSON decode error","input":{},"ctx":{"error":"Expecting property name enclosed in double quotes"}}]}`

Reply:
It should be fixed now